.gitlab-ci.yml

```diff
@@ -12,33 +12,28 @@ variables:
   SINGULARITY_EOS_TEMP_SPACK_VERSION: ""
   SINGULARITY_EOS_TEMP_SPACK_DIR: ""
   SINGULARITY_EOS_SPACK_SPEC: "singularity-eos@main+mpi+python+tests%gcc@${SINGULARITY_EOS_GCC_VERSION} ^openmpi@${SINGULARITY_EOS_OPENMPI_VERSION}"
+  COLOR_CYAN: '\033[1;36m'
+  COLOR_PLAIN: '\033[0m'
 
 before_script:
-  - export HOME=${CI_PROJECT_DIR}
   - export SPACK_DISABLE_LOCAL_CONFIG=true
-  - export SPACK_USER_CACHE_PATH=/tmp/spack-local
-  - cd ${CI_PROJECT_DIR}
-  - unset SPACK_ROOT
-  - export SINGULARITY_EOS_SPACK_VERSION=$([ -z "$SINGULARITY_EOS_TEMP_SPACK_VERSION" ] && echo $XCAP_OSS_SPACK_REF || echo $SINGULARITY_EOS_TEMP_SPACK_VERSION)
```

Comment:
Do we not want this logic to handle a different spack version when the occasion arises anymore?

Reply:
It's still there, but you need to specify the full directory path `SINGULARITY_EOS_TEMP_SPACK_DIR` instead.

Reply:
we can obviously make it so that there is still the "version" only that need to be modified, as long as the version matches the subdirectory name.

Reply:
The less that needs to be modified in this script when a spack version changes, the better IMO

Reply:
Ohh the `:=` lets the `SINGULARITY_EOS_TEMP_SPACK_DIR` through if it's not empty. Yeah, then it's fine either way with me really